src/handlers/jobs.rs

```diff
@@ -0,0 +1,21 @@
+// Function to match the scheduled job function with its corresponding handler.
+// In case you want to add a new one, just add a new clause to the match with 
+// the job name and the corresponding function.
+
+// The metadata is a serde_json::Value, please visit: https://docs.rs/serde_json/latest/serde_json/value/enum.Value.html
+// to refer on how to get values from there.
+// Example of accessing an integer id in the metadata:
+//    metadata["id"].as_i64().unwrap();
```

Comment:
We'll likely want to deserialize here into a dedicated struct for each handler which implements Deserialize. If you wanted to add a sample handler of part of this PR, one easy-ish option is one that sends a Zulip message every Friday at 11:30am ET into #t-release with a `@T-release meeting!` content. (Or something even more basic).

For that case, the metadata would just be a unit `()`.